load/loader_test.go

```diff
@@ -102,13 +102,13 @@ func TestGetBufferedReader(t *testing.T) {
 		t.Errorf("initial buffered reader is non-nil")
 	}
 	// TODO Filename not yet supported
-	r.filename = "foo"
+	r.fileName = "foo"
 	br = r.GetBufferedReader()
 	if br != nil {
 		t.Errorf("filename returned a non-nil buffered reader")
 	}
 	// Should give a non-nil bufio.Reader now
-	r.filename = ""
+	r.fileName = "/dev/null"
```

Comment:
I think this case is supposed to test whether reading from STDIN works, so I don't think you should change the case here.

Reply:
Agree, my mistake. Sure we need to test for STDIN